sdk/eventgrid/Azure.Messaging.EventGrid/tests/Infrastructure/EventGridLiveTestBase.cs

```diff
@@ -7,11 +7,16 @@ namespace Azure.Messaging.EventGrid.Tests
 {
     public class EventGridLiveTestBase : RecordedTestBase<EventGridTestEnvironment>
     {
-        public EventGridLiveTestBase(bool isAsync) : base(isAsync /*, RecordedTestMode.Record */)
+        // These constants should eventually go in our product code, as they will be needed when creating
+        // the AzureKeyCredential.
+        public const string SasKeyName = "aeg-sas-key";
```

Comment:
Once we do internals visible to, we can use the constants from the other project, and delete these.